modules/tinymce/src/themes/silver/main/ts/ui/context/ContextToolbar.ts

```diff
@@ -286,6 +288,16 @@ const register = (editor: Editor, registryContextToolbars: Record<string, Contex
       }
     });
 
+    editor.on('dragstart', () => {
+      isDragging = true;
+    });
+    editor.on('dragend', () => {
+      isDragging = false;
+    });
+    editor.on('drop', () => {
+      isDragging = false;
+    });
```

Comment:
Maybe this:

```suggestion
    editor.on('dragend drop', () => {
      isDragging = false;
    });
```

Or even but maybe harder to read:
```
    editor.on('dragstart dragend drop', (e) => {
      isDragging = e.type === 'dragstart';
    });
```

Reply:
I just merged `dragend` and `drop`, I think merging all 3 events now, is no needed.

If in the future we add other behaviors across all these events maybe it can become something like:
```
const setIsDragging = (eventType) => {
   isDragging = eventType === 'dragstart';
}
editor.on('dragstart dragend drop', (e) => {
   setIsDragging(e.type);
});
```
but by now have 2 handlers seems more clear